docs/guides/storage.md

```diff
@@ -67,6 +66,28 @@ pip install torch torchvision torchaudio \
 
     In our "simple" pytorch example, I counted **22806 inodes**!
 
+##### Alternative virtual environment creation using uv
```

Comment:
Instead of adding the `uv` method as a sub-heading under the `pip` method, how about putting them at the same level as tabs?

With one tab for `pip`, and one for `uv` (I would be fine with making the "first" default tab the `uv` one, so that we can push users towards `uv`.

https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#usage

Reply:
See here for an example of what tabs look like in our docs
https://eth-cscs.github.io/cscs-docs/software/tools/linaro/#quickstart-guide